<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesDock</class>
 <widget class="QWidget" name="NotesDockGeneralTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>165</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lName">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="KLineEdit" name="leName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lComment">
     <property name="text">
      <string>Comment</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="KLineEdit" name="leComment"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="bgcolor">
     <property name="text">
      <string>Backgorund Color</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="KColorButton" name="kcbBgColor" />
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="textcolor">
     <property name="text">
      <string>Text Color</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="KColorButton" name="kcbTextColor" />
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacerGeneral">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>24</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
